tasks/vm_state_present.yml

```diff
@@ -30,6 +30,7 @@
     bootable: "{{ item.1.bootable | default(omit) }}"
     activate: "{{ item.1.activate | default(omit) }}"
     wait: true
+  # When there is no profile disks it will use vms disks
```

Comment:
This is not true, the conditional which is used later (when: item.0.profile is defined and item.0.profile.disks is defined) actually succed only if some profile disk exists.